packages/gestalt/src/SheetMobile.js

```diff
@@ -150,7 +145,6 @@ function SheetMobile({
   if (size === 'full')
     return (
       <FullPage
-        accessibilityLabel={accessibilityLabel}
```

Comment:
Should this prop be removed from the FullPage and PartialPage components as well?

Reply:
@dangerismycat Left it there for now, as I might try to reuse this for Modal, and modal do has an accessibility label.